lib/puppet/util/run_mode.rb

```diff
@@ -38,6 +38,10 @@ def log_dir
         "$vardir/log"
       end
 
+      def conf_dir
+        "$puppetdir/config"
+      end
+
```

Comment:
On Wed, Feb 04, 2015 at 12:09:16PM -0800, Chris Price wrote:
> > @@ -38,6 +38,10 @@ def log_dir
> >          "$vardir/log"
> >        end
> >  
> > +      def conf_dir
> > +        "$puppetdir/config"
> > +      end
> > +
> 
> Then we should probably just remove the `log_dir` one, because if it were ever used, it would have the same problem... right?

It is used but not directly like this :-)


-- 
 (o-    Julien Pivotto
 //\    Open-Source Consultant
 V_/_   Inuits - https://www.inuits.eu